lms/templates/main.html

```diff
@@ -84,31 +94,7 @@
 
   <%block name="headextra"/>
 
-<%
-  if theme_enabled() and not is_microsite():
-    header_extra_file = 'theme-head-extra.html'
-    header_file = 'theme-header.html'
-    google_analytics_file = 'theme-google-analytics.html'
-
-    style_overrides_file = None
-
-  else:
-    header_extra_file = microsite.get_template_path('header_extra.html')
-
-    if settings.FEATURES['IS_EDX_DOMAIN'] and not is_microsite():
-        header_file = microsite.get_template_path('navigation-edx.html')
```

Comment:
@andy-armstrong Yes, I think the template can be removed as it is no longer needed.  Usage was removed when a new theme was created named `edx.org` instead of using `IS_EDX_DOMAIN` flag. edx.org theme contains all the overrides in [`heade.html`](https://github.com/edx/edx-platform/blob/master/themes/edx.org/lms/templates/header.html) that were previously contained in `navigation-edx.html `. 
@ziafazal FYI